### ISSUE TITLE
Photo tiles missing Alt

### ISSUE THREAD

**Issue by naher94** [naher94/rehanbutt.com#601]
on `/photography` the tiles are missing real alt text

**Comment by naher94** [comment]
Burj al Arab also missing alt text on the `https://rehanbutt.com/photography/assorted-snaps.html`

likely missing from the front matter
